metas.groovy

```diff
@@ -0,0 +1,137 @@
+//Adapted from module.groovy
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group='org.ajoberstar', module='grgit', version='1.9.3')
+import org.ajoberstar.grgit.Grgit
+import groovy.json.JsonSlurper
+
+Properties properties = new Properties()
+new File("gradle.properties").withInputStream {
+    properties.load(it)
+}
+
+githubHome = properties.alternativeGithubHome ?: "MetaTerasology"
+
+modulesRetrieved = []
+
+
+def retrieve(String[] modules) {
+    println "Now inside retrieve, user wants: $modules"
+    for (String module : modules) {
+        println "Starting loop for meta module $module"
+        println "Meta Modules retrieved so far: $modulesRetrieved"
+        retrieveModule(module)
+    }
+}
+
+def retrieveModule(String module) {
+    File targetDir = new File("meta/$module")

```

Comment:
This needs a little more. Again check the original tasks in `utility.gradle` :-)

Specifically:

```
            // Since meta-modules only accompany normal modules they need a unique name
            def destination = file("$parentDir/$repo" + "Meta")
```

Reply:
whoops.... will fix that